src/spatial.jl

```diff
@@ -27,6 +27,11 @@ Setting one or more values to `Inf` gives no boundary in that dimension.
 """
 struct CubicBoundary{T}
     side_lengths::SVector{3, T}
+    CubicBoundary(side_lengths::SVector{3, E}) where E = (
```

Comment:
This could be written in the standard function form, i.e. `function`, for readability. Also with an explicit `if`/`else`.

Reply:
Indeed, I have made the change. Thanks for pointing it out.